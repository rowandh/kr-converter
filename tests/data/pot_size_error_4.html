
<iframe width="0" height="0" name="hfrm"></iframe>
<script language="javascript">
	function goProc(mode,uid,infotype,money){
		form = document.form1;
		form.target="hfrm";
		if (money < 1) {
			alert("1보다 큰금액만 입력하실수 있습니다");
			return;
		}
		form.infotype.value=infotype;
		form.money.value=money;
		form.user_id.value=uid;
		form.action="userinmoney_proc.asp";
		
			if(confirm(uid + "에게 " + money + "원 하자처리 하시겠습니까?")){
				form.submit();
			}		

	}
</script>

<style>
		body {background:#f5f5f5;}
		* {padding:0; margin:0;}
		input:focus,
		select:focus {
			outline:none;
		}
		.page-layout {
			height:100%;
			
		}
		
		.page-layout .bar-search {
			display:flex;
			justify-content: space-between;
			align-items: center;
			flex-wrap: wrap;
			gap:0.875rem;
			
			padding:0 0.875rem 0.875rem;
			background:#2e2f39;
		    border-bottom: 1px solid #e6e6f2;
		    box-shadow: 0 1px 4px 0 rgba(82,63,105,.15);
		}
		
		.page-layout .bar-search .title {
			padding-top:0.875rem;
			font-family:'Malgun Gothic';
			color:#c5ff88;
			font-size:1.425rem;
			font-weight:bolder;
			letter-spacing:-2px;
		}
		
		.page-layout .bar-search .title > span {
			font-size:1rem;
			letter-spacing:0;
		}
		
		.page-layout .bar-search > div:nth-child(2) {
			flex-grow:1;
			
			display:inline-flex;
    		flex-wrap: wrap;
    		justify-content: flex-end;
			gap:0.875rem;
			
			padding-top:0.875rem;
			border-top:1px solid #60931e;
		}
		
		.page-layout .bar-search > div:nth-child(2) span {
		    line-height:2rem;
		    color: #fff;
		    font-weight:bold;
		}
		
		.page-layout .bar-search > div:nth-child(2) select,
		.page-layout .bar-search > div:nth-child(2) input {
		    height: calc(2em + 0.8rem + 2px);
		    padding:0 1rem;
		    font-size: 1rem;
		    font-weight: 400;
		    height:2rem;
		    line-height:2rem;
		    color: #ccc;
		    background-color: #141529;
		    background-clip: padding-box;
		    border: 1px solid #141529;
		    border-radius: 0.25rem;
		    transition: border-color .15s ease-in-out,box-shadow .15s ease-in-out;
		}
		
		.page-layout .bar-search > div:nth-child(2) input.btn-serarch {
			color:#fff;
			font-weight:bold;
			background:#73a730 ;
			border:1px solid #629322;
		}
		
		
		.table-area {
			border:1px solid #e6e6f2;
			border-radius:1rem;
			overflow-x:auto;
			background:#fff;
			box-shadow:3px 3px 5px #e7e7e7;
			margin:1rem;
		}
		
		.table-area th {
			padding: 0 0.5rem;
			font-size:1rem;
			height:46px;
			font-weight: 600;
    		color: #2e2f39;
			background:#fff;
			border-bottom:2px solid #e6e6f2;
		}
		
		.table-area th a {
			color:#cf0b0b;
			text-decoration:none;
		}
		
		.table-area td {
			font-size:14px;
			padding:0.425rem 0.75rem;
			color:#71748d;
			white-space:nowrap;
			background:#fff;
			border-bottom:1px solid #e6e6f2;
		}
		/*
		.table-area tr:nth-child(2n) td {
			background:#f4f4f5;
			border-top:1px solid #e6e6f2;
			border-bottom:1px solid #e6e6f2;
		}
		*/
		.table-area th,
		.table-area td {
			border-right:1px solid #e6e6f2;
		}
		
		.table-area table td a {
			display:inline-block;
			
			padding:0 1rem 0 1.425rem;
			line-height:1rem;
			color:#5e8d21;
			font-size:14px;
			font-weight:bold;
			text-decoration:none;
			border-radius:0.312rem;
			background:url('../images/icon-move.png') left 50% no-repeat;
		}
		
		.table-area table td a:hover {
			filter:hue-rotate(275deg);
		}
		
		.table-area table th.double-table,
		.table-area table td.double-table {
			padding:0;
		}
		
		
		.table-area table th.double-table th {
			padding:0 0.75rem;
			border-bottom:none;
		}
		
		.table-area table td.double-table td {
			line-height:1.5rem;
			border:none;
			background:none;
		}
		
		.table-area table td.double-table tr td+td {
			border-left:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr+tr td {
			border-top:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr:nth-child(2n) td {
			background:#f9f9f9;
		}
		
		.btn-history-back {
			padding:0 1rem;
			font-size:1rem;
			height:2.625rem;
			line-height:2.625rem;
			color:#fff;
			font-weight:bold; 
			background:#37a730; 
			border-radius:0.25rem; 
			border:1px solid #629322; 
			box-shadow:1px 1px 0 #629322;
			cursor:pointer;
		}
		
		.btn-history-back:hover {
			filter:brightness(1.1);
			border:1px solid #37a730; 
			box-shadow:1px 1px 0 #37a730;
		}
		
	</style>
	
	<div class="page-layout">
		<div class="bar-search">
			<div class="title">게임기록보기</div>
			<div>
                <div>
					<input type="button" value="닫기" onclick="self.close()" class="btn-serarch" />
                </div>	
			</div>
		</div>
		
		<div class="table-area">
			<table cellspacing="0" width="100%"  border="0" >
				<tr>
                          <th width="" height="34"><p align="center">라운드ID</p></th>
                          <th width="" height="34"><p align="center">시각</p></th>
                          <th width="" height="34"><p align="center">게임 종류</p></th>
                          <th width="" height="34"><p align="center">승자(족보)</p></th>
					<th width="" height="34"><p align="center">이긴금액</p></th>
					<th class="double-table">
						상세정보
					</th>
				</tr>




                            <tr bgcolor="white">
								<td width="56" height="26"><p align="center">&nbsp;15-3-29520969</p></td>
                                <td height="26" ><p align="center">&nbsp;2020-10-15 오후 6:50:59</p></td>
                                <td ><p align="center">&nbsp;홀덤</p></td>
                                <td ><p align="center">&nbsp;<b>etrbojng (Q 트리플)</b></p></td>
<!--								<td ><p align="center">&nbsp;964151</p></td>   -->
								<td ><p align="center">&nbsp;964,151</p></td>
								<td class="double-table">
									
									<table cellspacing="0" width="100%"  border="0">										
										<colgroup>
											<col width="200" />
											<col width="*" />
											<col width="160" />
											<col width="200" />
										</colgroup>
										<tr>
											<th height="34" style="word-break:break-all"><p align="center">참가자</p></th>
											<th height="34" style="word-break:break-all"><p align="center">족보</p></th>
											<th height="34" style="word-break:break-all"><p align="center">변동 금액</p></th>
											<th height="34" style="word-break:break-all"><p align="center">남은 잔액</p></th>
										</tr>
										
										


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>keiserturn</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣J ♣6 [J 탑]  (다이) 
<br>* 시작 : [StageNo:29520969] [Credit:722,862원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:300]
<br>* NICKNAME:[keiserturn]
<br>* 앤티: -5,000원(717,862원)
<br>* 홀 카드딜: ♣J(49) ♣6(44) [J 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:J 탑(♣J ♣6)]
<br>* 베팅: 다이 [0](717,862원) - 베팅순서: [0][5]
[2353ms]
<br>* 커뮤니티 카드 딜: H(♣J♣6) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q)
<br>* 방이동요청[0]
<br>* 방나감

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-5,000</font></p></td>
											<td style="word-break:break-all"><p align="right">717,862</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>coskcjss</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠9 <font color='red'>♥</font>9 <font color='red'>◆</font>2 <font color='red'>◆</font>9 ♠Q ♣5 <font color='red'>♥</font>7 [9 트리플]  
<br>* 시작 : [StageNo:29520969] [Credit:1,786,760원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:100]
<br>* NICKNAME:[coskcjss]
<br>* 앤티: -5,000원(1,781,760원)
<br>* 홀 카드딜: ♠9(8) <font color='red'>♥</font>9(34) [9 원 페어]
<br>* 턴 시작: [프리플랍(0)] [족보:9 원 페어(♠9 <font color='red'>♥</font>9)]
<br>* 베팅: [레이즈] (40,000원) Credit(1,741,760원) - 베팅순서: [0][3]
[1327ms]
<br>* 커뮤니티 카드 딜: H(♠9<font color='red'>♥</font>9) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q)
<br>* 턴 시작: [플랍(1)] [족보:9 트리플(<font color='red'>◆</font>9 ♠9 <font color='red'>♥</font>9 ♠Q <font color='red'>◆</font>2)]
<br>* 베팅: 콜 -135,000원(1,606,760원) - 베팅순서: [1][2]
[4259ms]
<br>* 커뮤니티 카드 딜: H(♠9<font color='red'>♥</font>9) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q) (♣5)
<br>* 턴 시작: [턴(2)] [족보:9 트리플(<font color='red'>◆</font>9 ♠9 <font color='red'>♥</font>9 ♠Q ♣5)]
<br>* 베팅: [풀] (630,000원) Credit(976,760원) - 베팅순서: [2][2]
[3394ms]
<br>* 쇼다운 상태 진입: [턴:2]
<br>* 커뮤니티 카드 딜: H(♠9<font color='red'>♥</font>9) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q) (♣5) (<font color='red'>♥</font>7)
# 공베팅 반환 [323,445원]<br>* 종료: WinMoney[0원] Credit[1,300,205원]
<br>* 결과: 패배 [족보:9 트리플] [카드:<font color='red'>◆</font>9 ♠9 <font color='red'>♥</font>9 ♠Q <font color='red'>♥</font>7]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-486,555</font></p></td>
											<td style="word-break:break-all"><p align="right">1,300,205</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>dxbbtrff</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>A ♣3 [A 탑]  (다이) 
<br>* 시작 : [StageNo:29520969] [Credit:500,000원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:100]
<br>* NICKNAME:[dxbbtrff]
<br>* 앤티: -5,000원(495,000원)
<br>* 홀 카드딜: <font color='red'>◆</font>A(13) ♣3(41) [A 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:A 탑(<font color='red'>◆</font>A ♣3)]
<br>* 베팅: 다이 [0](495,000원) - 베팅순서: [0][6]
[490ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>A♣3) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>A♣3) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q) (♣5)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>A♣3) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q) (♣5) (<font color='red'>♥</font>7)
<br>* 종료: WinMoney[0원] Credit[495,000원]
<br>* 결과: 패배 [족보:A 탑] [카드:<font color='red'>◆</font>A ♣3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-5,000</font></p></td>
											<td style="word-break:break-all"><p align="right">495,000</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>etrbojng</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>Q <font color='red'>◆</font>Q <font color='red'>◆</font>2 <font color='red'>◆</font>9 ♠Q ♣5 <font color='red'>♥</font>7 [Q 트리플]  (Big Blind)
<br>* 시작 : [StageNo:29520969] [Credit:486,555원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:100]
<br>* NICKNAME:[hfngxymk]
<br>* 앤티: -5,000원(481,555원)
<br>* 홀 카드딜: <font color='red'>♥</font>Q(37) <font color='red'>◆</font>Q(24) [Q 원 페어]
<br>* 턴 시작: [프리플랍(0)] [족보:Q 원 페어(<font color='red'>♥</font>Q <font color='red'>◆</font>Q)]
<br>* 베팅: [블라인드:BIG] [금액:5,000원] [Creadit:476,555원]
<br>* 베팅: 콜 -35,000원(441,555원) - 베팅순서: [0][9]
[1168ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>Q<font color='red'>◆</font>Q) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q)
<br>* 턴 시작: [플랍(1)] [족보:Q 트리플(♠Q <font color='red'>♥</font>Q <font color='red'>◆</font>Q <font color='red'>◆</font>9 <font color='red'>◆</font>2)]
<br>* 베팅: [풀] (135,000원) Credit(306,555원) - 베팅순서: [1][1]
[3766ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>Q<font color='red'>◆</font>Q) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q) (♣5)
<br>* 턴 시작: [턴(2)] [족보:Q 트리플(♠Q <font color='red'>♥</font>Q <font color='red'>◆</font>Q <font color='red'>◆</font>9 ♣5)]
<br>* 베팅: [베팅] (75,000원) Credit(231,555원) - 베팅순서: [2][1]
[5103ms]
<br>* 베팅: 콜 -231,555원(0원) - 베팅순서: [2][3]
[2173ms]
<br>* 쇼다운 상태 진입: [턴:2]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>Q<font color='red'>◆</font>Q) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q) (♣5) (<font color='red'>♥</font>7)
<br>* 종료: WinMoney[964,151원] Credit[964,151원]
<br>* 결과: 승리 [족보:Q 트리플] [카드:♠Q <font color='red'>♥</font>Q <font color='red'>◆</font>Q <font color='red'>◆</font>9 <font color='red'>♥</font>7]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>477,596</font></p></td>
											<td style="word-break:break-all"><p align="right">964,151</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>susbdj</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>K ♠4 [K 탑]  (다이) 
<br>* 시작 : [StageNo:29520969] [Credit:655,193원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:300]
<br>* NICKNAME:[susbdj]
<br>* 앤티: -5,000원(650,193원)
<br>* 홀 카드딜: <font color='red'>♥</font>K(38) ♠4(3) [K 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:K 탑(<font color='red'>♥</font>K ♠4)]
<br>* 베팅: 다이 [0](650,193원) - 베팅순서: [0][1]
[2200ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>K♠4) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q)
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>K♠4) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q) (♣5)
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>K♠4) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q) (♣5) (<font color='red'>♥</font>7)
<br>* 종료: WinMoney[0원] Credit[650,193원]
<br>* 결과: 패배 [족보:K 탑] [카드:<font color='red'>♥</font>K ♠4] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-5,000</font></p></td>
											<td style="word-break:break-all"><p align="right">650,193</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>vjghojtks</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>J <font color='red'>♥</font>2 [J 탑]  (다이) 
<br>* 시작 : [StageNo:29520969] [Credit:601,600원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:100]
<br>* NICKNAME:[vjghojtks]
<br>* 앤티: -5,000원(596,600원)
<br>* 홀 카드딜: <font color='red'>◆</font>J(23) <font color='red'>♥</font>2(27) [J 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:J 탑(<font color='red'>◆</font>J <font color='red'>♥</font>2)]
<br>* 베팅: 다이 [0](596,600원) - 베팅순서: [0][4]
[6695ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>J<font color='red'>♥</font>2) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>J<font color='red'>♥</font>2) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q) (♣5)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>J<font color='red'>♥</font>2) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q) (♣5) (<font color='red'>♥</font>7)
<br>* 종료: WinMoney[0원] Credit[596,600원]
<br>* 결과: 패배 [족보:J 탑] [카드:<font color='red'>◆</font>J <font color='red'>♥</font>2] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-5,000</font></p></td>
											<td style="word-break:break-all"><p align="right">596,600</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>jhnghutzh</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣A ♠5 [A 탑]  (다이) 
<br>* 시작 : [StageNo:29520969] [Credit:1,577,280원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:200]
<br>* NICKNAME:[jhnghutzh]
<br>* 앤티: -5,000원(1,572,280원)
<br>* 홀 카드딜: ♣A(39) ♠5(4) [A 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:A 탑(♣A ♠5)]
<br>* 베팅: 다이 [0](1,572,280원) - 베팅순서: [0][7]
[863ms]
<br>* 커뮤니티 카드 딜: H(♣A♠5) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q)
<br>* 커뮤니티 카드 딜: H(♣A♠5) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q) (♣5)
<br>* 커뮤니티 카드 딜: H(♣A♠5) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q) (♣5) (<font color='red'>♥</font>7)
<br>* 종료: WinMoney[0원] Credit[1,572,280원]
<br>* 결과: 패배 [족보:A 탑] [카드:♣A ♠5] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-5,000</font></p></td>
											<td style="word-break:break-all"><p align="right">1,572,280</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>Pjeibebdn</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>J <font color='red'>♥</font>4 [J 탑]  (다이) 
<br>* 시작 : [StageNo:29520969] [Credit:623,758원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:100]
<br>* NICKNAME:[pjeibebdn]
<br>* 앤티: -5,000원(618,758원)
<br>* 홀 카드딜: <font color='red'>♥</font>J(36) <font color='red'>♥</font>4(29) [J 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:J 탑(<font color='red'>♥</font>J <font color='red'>♥</font>4)]
<br>* 베팅: 콜 -5,000원(613,758원) - 베팅순서: [0][2]
[308ms]
<br>* 베팅: 다이 [0](613,758원) - 베팅순서: [0][10]
[201ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>J<font color='red'>♥</font>4) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q)
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>J<font color='red'>♥</font>4) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q) (♣5)
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>J<font color='red'>♥</font>4) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q) (♣5) (<font color='red'>♥</font>7)
<br>* 종료: WinMoney[0원] Credit[613,758원]
<br>* 결과: 패배 [족보:J 탑] [카드:<font color='red'>♥</font>J <font color='red'>♥</font>4] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-10,000</font></p></td>
											<td style="word-break:break-all"><p align="right">613,758</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>85xcdkfoqs</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>5 ♣4 [5 탑]  (다이) (Small Blind)
<br>* 시작 : [StageNo:29520969] [Credit:1,500,000원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:300]
<br>* NICKNAME:[85xcdkfoqs]
<br>* 앤티: -5,000원(1,495,000원)
<br>* 홀 카드딜: <font color='red'>◆</font>5(17) ♣4(42) [5 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:5 탑(<font color='red'>◆</font>5 ♣4)]
<br>* 베팅: [블라인드:SMALL] [금액:5,000원] [Creadit:1,490,000원]
<br>* 베팅: 다이 [0](1,490,000원) - 베팅순서: [0][8]
[2501ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>5♣4) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>5♣4) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q) (♣5)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>5♣4) C (<font color='red'>◆</font>2<font color='red'>◆</font>9♠Q) (♣5) (<font color='red'>♥</font>7)
<br>* 종료: WinMoney[0원] Credit[1,490,000원]
<br>* 결과: 패배 [족보:5 탑] [카드:<font color='red'>◆</font>5 ♣4] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-10,000</font></p></td>
											<td style="word-break:break-all"><p align="right">1,490,000</p></td>
											
										</tr>

									 </table>
								</td>
                            </tr>






						
                        </table>
                        
                        <table cellpadding="0" cellspacing="0" border="0" width="100%">
                            <tr>
                                <td width="100%" align="center" style="padding:1rem 0; background:#fdfdfd;">
                                	<input type="button" value="닫기" onclick="self.close()" class="btn-history-back">
                                </td>
                            </tr>
                        </table>
                        
		</div>
                        
	</div>




<form name="form1" method="Post">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="logno" value="15-3-29520969">
<input type="hidden" name="regdate_f" value="">
<input type="hidden" name="user_id">
<input type="hidden" name="userid">
<input type="hidden" name="rebateid">
<input type="hidden" name="topid">
<input type="hidden" name="infotype">
<input type="hidden" name="money">
</form>

<form name="form3" method="post">
<input type="hidden" name="idx">
<input type="hidden" name="p_logno">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="page" value="">

<input type="hidden" name="s_regdate_y1" value="">
<input type="hidden" name="s_regdate_m1" value="">
<input type="hidden" name="s_regdate_d1" value="">
<input type="hidden" name="s_regdate_h1" value="">
<input type="hidden" name="game_id" value="">
<input type="hidden" name="s_regdate" value="">
</form>


