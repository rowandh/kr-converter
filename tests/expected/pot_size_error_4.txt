PokerStars Hand #15329520969:  Hold'em No Limit ($5000/$5000) - 2020/10/15 18:50:59 KST
Table 'Table 1' 9-max Seat #7 is the button
Seat 1: susbdj ($655193 in chips)
Seat 2: Pjeibebdn ($623758 in chips)
Seat 3: coskcjss ($1786760 in chips)
Seat 4: vjghojtks ($601600 in chips)
Seat 5: keiserturn ($722862 in chips)
Seat 6: dxbbtrff ($500000 in chips)
Seat 7: jhnghutzh ($1577280 in chips)
Seat 8: 85xcdkfoqs ($1500000 in chips)
Seat 9: etrbojng ($486555 in chips)
susbdj: posts the ante $5000
Pjeibebdn: posts the ante $5000
coskcjss: posts the ante $5000
vjghojtks: posts the ante $5000
keiserturn: posts the ante $5000
dxbbtrff: posts the ante $5000
jhnghutzh: posts the ante $5000
85xcdkfoqs: posts the ante $5000
etrbojng: posts the ante $5000
85xcdkfoqs: posts small blind $5000
etrbojng: posts big blind $5000
*** HOLE CARDS ***
susbdj: folds
Pjeibebdn: calls $5000
coskcjss: raises $35000 to $40000
vjghojtks: folds
keiserturn: folds
dxbbtrff: folds
jhnghutzh: folds
85xcdkfoqs: folds
etrbojng: calls $35000
Pjeibebdn: folds
*** FLOP *** [2d 9d Qs]
etrbojng: bets $135000
coskcjss: calls $135000
*** TURN *** [2d 9d Qs] [5c]
etrbojng: bets $75000
coskcjss: raises $555000 to $630000
etrbojng: calls $231555
Uncalled bet ($323445) returned to coskcjss
*** RIVER *** [2d 9d Qs] [5c] [7h]
etrbojng collected $964151 from pot
*** SUMMARY ***
Total pot $1018110 | Rake $53959
Board [2d 9d Qs 5c 7h]
Seat 1: susbdj mucked [Kh 4s]
Seat 2: Pjeibebdn mucked [Jh 4h]
Seat 3: coskcjss showed [9s 9h] and lost
Seat 4: vjghojtks mucked [Jd 2h]
Seat 5: keiserturn mucked [Jc 6c]
Seat 6: dxbbtrff mucked [Ad 3c]
Seat 7: jhnghutzh mucked [Ac 5s]
Seat 8: 85xcdkfoqs mucked [5d 4c]
Seat 9: etrbojng showed [Qh Qd] and won ($964151)